recipes/makeheaders/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "2018.1.28" %}
+{% set sha256 = "5eeb2b83357a4affefcf48a9d5624c1373a0cfd40d026306d1c2933888a145b0" %}
+{% set commit = "2d5bd5f81a6484c9d6af2542f83bf1f10c7c4f62" %}
+
+package:
+  name: makeheaders-rowanthorpe
+  version: {{ version }}
+
+source:
+  fn: makeheaders-{{ version }}.src.tar.xz
+  url: https://github.com/rowanthorpe/makeheaders/archive/{{ commit }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - gh_1.patch
+
+build:
+  number: 0
+  skip: true  # [win and py<35]
+  features:
+    - vc14    # [win and py>=35]
+
+requirements:
+  build:
+    - toolchain
+    - cmake >=3.4.3
+    - ninja    # [win]
+    - vc 14    # [win]
+  run:
+    - vc 14    # [win]
+
+test:
+  requires:
+    - python {{ environ['PY_VER'] + '*' }}  # [win]
+    - cmake >=3.4.3
+
+  commands:
+    - makeheaders
+
+about:
+  home: https://github.com/rowanthorpe/makeheaders
+  license: MIT
```

Comment:
It's actually BSD-2-Clause. And I think we need a `license_file`; since the repo doesn't have one, you should maybe manually package the comment from [the source file](https://github.com/rowanthorpe/makeheaders/blob/master/makeheaders.c#L6).